pytorch_lightning/trainer/trainer.py

```diff
@@ -408,36 +409,13 @@ def __init__(
         # Callback system
         self.on_init_end()
 
-    def fit(
+    def _launch(
```

Comment:
Nit : I prefer _run or _execute vs launch, which I associate more with distributed launch/process creation

Reply:
Bit late as it is already merged, but if you want to open a PR with the renaming change, that's fine by me! 👍